modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -0,0 +1,45 @@
+[id="creating-a-new-client-in-keycloak_{context}"]
+= Creating a new client in Keycloak
+
+.Procedure
+
+. Create a new Argo CD Instance in the `argocd` namespace
```

Comment:
This is not correct. It should look something like below.

1. Configure keycloak.
-> Edit the Argo CD CR and update `.sso.spec.provider: keycloak`